### PR TITLE
refactor Create order

### PR DESCRIPTION
**Pull request by MaxAstin**
- migrate to `SharedStateViewModel`
- migrate to `BaseComposeFragment`
- migrate to new Bottom Sheets

### CODE REVIEW

File: app/src/main/java/com/bunbeauty/papakarlo/common/ui/element/simmer/Shimmer.kt [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import androidx.compose.ui.draw.drawWithContent
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.TileMode
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.bunbeauty.papakarlo.common.ui.theme.FoodDeliveryTheme
+
+@Composable
+fun Shimmer(
```
Comment (xidealo) [inline app/src/main/java/com/bunbeauty/papakarlo/common/ui/element/simmer/Shimmer.kt:22]: А куда добавились шимеры?

Comment (MaxAstin) [reply app/src/main/java/com/bunbeauty/papakarlo/common/ui/element/simmer/Shimmer.kt:22]: Bottom panel on Create order

File: shared/src/commonMain/kotlin/com/bunbeauty/shared/presentation/createorder/CreateOrderViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,482 @@
...
+        withLoading {
+            updatePaymentMethods()
+        }
+        withLoading {
+            updateCartTotal()
+        }
+    }
+
+    private fun changeMethod(position: Int) {
+        (position == 0).let { isDelivery ->
```
Comment (xidealo) [inline shared/src/commonMain/kotlin/com/bunbeauty/shared/presentation/createorder/CreateOrderViewModel.kt:155]: magic number

**Review by xidealo — APPROVED** [review]

File: shared/src/commonMain/kotlin/com/bunbeauty/shared/presentation/createorder/CreateOrderViewModel.kt (hunk 2 above)
Comment (MaxAstin) [reply shared/src/commonMain/kotlin/com/bunbeauty/shared/presentation/createorder/CreateOrderViewModel.kt]: fixed

**Pull request merged**